### PR TITLE
[PR] The proj files have been updated to enable SourceLink

### PR DESCRIPTION
**Pull request by JTOne123**
CSProj files have been updated to enable SourceLink in your nuget
---

*[This pull request was created with an automated workflow]*

I noticed that your repository and Nuget package are important for our .NET community, but you still haven't enabled SourceLink.

**We have to take 2 steps:**
1) Please approve this pull request and make .NET a better place for .NET developers and their debugging.
2) **Then just upload the .snupkg file** to https://www.nuget.org/ (now you can find the snupkg file along with the .nuget file)

You can find more information about SourceLine at the following links  
https://github.com/dotnet/sourcelink
https://www.hanselman.com/blog/ExploringNETCoresSourceLinkSteppingIntoTheSourceCodeOfNuGetPackagesYouDontOwn.aspx

If you are interesting about this automated workflow and how it works  
https://github.com/JTOne123/GitHubMassUpdater

*If you notice any flaws, please comment and I will try to make fixes manually*

### CODE REVIEW

File: src/IF.Lastfm.Core/IF.Lastfm.Core.csproj [hunk 1]
```diff
@@ -3,8 +3,16 @@
   <PropertyGroup>
     <Version />
     <PackageVersion />
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```
Comment (rikkit) [inline src/IF.Lastfm.Core/IF.Lastfm.Core.csproj:12]: Could this entry be merged with the ItemGroup on L20? Sure it doesn't matter, just curious.

**Comment by rikkit** [comment]
Thanks @JTOne123 !

Ideally the `.snupkg` file would be automatically published on builds. I need to fix the build pipeline anyway, which is something I'm looking into.

**Comment by JTOne123** [comment]
https://www.meziantou.net/how-to-debug-nuget-packages-using-sourcelink.htm

**Comment by rikkit** [comment]
Hi @JTOne123, I think I have done this as part of #159. [Symbols for v1.2.0 are on NuGet](https://www.nuget.org/packages/Inflatable.Lastfm/1.2.0). Thanks!

**Pull request closed without merging**